Ds3/Helpers/Ds3ClientHelpers.cs

```diff
@@ -111,27 +110,21 @@ private static long GetJobSize(IEnumerable<Ds3Object> objectsToWrite)
         }
 
         public IJob StartReadJob(string bucket, IEnumerable<Ds3Object> objectsToRead,
-            IHelperStrategy<string> helperStrategy = null)
+            IHelperStrategy<string> helperStrategy = null, Ds3ReadJobOptions ds3ReadJobOptions = null)
```

Comment:
Same comment here on using a default options object rather than setting it to null.